src/toil_vg/vg_evaluation_pipeline.py

```diff
@@ -204,6 +216,142 @@ def run_pipeline_merge_vcf(job, options, vcf_tbi_file_id_pair_list):
     write_to_store(job, options, vcf_file, use_out_store = True, out_store_key = out_store_key)
     write_to_store(job, options, vcf_file_idx, use_out_store = True, out_store_key = out_store_key + ".tbi") 
 
+def generate_config():
```

Comment:
I've thought about modularizing the config file generation in a way that's similar to what you did with adding arguments to the argument parser. It might be possible using yaml tools. I can look into it further.